### PR TITLE
docs: Make MacOS user path consistent across docs

### PR DESCRIPTION
**Pull request by RandyMcMillan**

### CODE REVIEW

**Comment by RandyMcMillan** [comment]
```bash
$ git grep "/Users"
docs/INSTALL.md:      --btcd.dir=                                             The base directory that contains the node's data, logs, configuration file, etc. (default: /Users/[username]/Library/Application Support/Btcd)
docs/INSTALL.md:      --btcd.rpccert=                                         File containing the daemon's certificate file (default: /Users/[username]/Library/Application Support/Btcd/rpc.cert)
docs/INSTALL.md:      --bitcoind.dir=                                         The base directory that contains the node's data, logs, configuration file, etc. (default: /Users/[username]/Library/Application Support/Bitcoin)
docs/INSTALL.md:`/Users/[username]/Library/Application Support/Lnd/lnd.conf`
docs/grpc/java.md:  private static final String CERT_PATH = "/Users/[username]/Library/Application Support/Lnd/tls.cert";
docs/grpc/java.md:  private static final String MACAROON_PATH = "/Users/[username]/Library/Application Support/Lnd/data/chain/bitcoin/simnet/admin.macaroon";
docs/grpc/java.md:[INFO] Compiling 3 proto file(s) to /Users/[username]/Documents/Projects/lightningclient/target/generated-sources/protobuf/java
docs/grpc/java.md:[INFO] Compiling 3 proto file(s) to /Users/[username]/Documents/Projects/lightningclient/target/generated-sources/protobuf/grpc-java
docs/grpc/java.md:[INFO] Compiling 12 source files to /Users/[username]/Documents/Projects/lightningclient/target/classes

File: docs/INSTALL.md [hunk 1]
```diff
@@ -316,11 +316,11 @@ The set of arguments for each of the backend modes is as follows:
 ## btcd Options
 ```text
 btcd:
-      --btcd.dir=                                             The base directory that contains the node's data, logs, configuration file, etc. (default: /Users/roasbeef/Library/Application Support/Btcd)
+      --btcd.dir=                                             The base directory that contains the node's data, logs, configuration file, etc. (default: /Users/[username]/Library/Application Support/Btcd)
```
[the remark this answers is not in the document]
Comment (RandyMcMillan) [reply docs/INSTALL.md:319]: I agree

[the remark this answers is not in the document]
Comment (RandyMcMillan) [reply docs/INSTALL.md:319]: will <...> break any automated doc generation? I assumed it might - that is why I used the established convention. 😄

**Comment by RandyMcMillan** [comment]
2b7c901 changed to \<username\>
ed94a8f changed to \<username\>

File: docs/INSTALL.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (RandyMcMillan) [reply docs/INSTALL.md]: updated per your suggestion...

**Review by guggero — APPROVED** [review]

**Review by yyforyongyu — APPROVED** [review]

**Pull request merged**